app/assets/javascripts/app/models/post/interactions.js

```diff
@@ -44,12 +44,16 @@ app.models.Post.Interactions = Backbone.Model.extend({
   },
 
   userLike : function(){
-    return this.likes.select(function(like){ return like.get("author").guid === app.currentUser.get("guid")})[0];
+    return this.likes.select(function(like){
+      if(like.get("author") !== undefined) {
```

Comment:
@Flaburgan We do the same for reshares:

```javascript
return reshare.get("author") && reshare.get("author").guid === app.currentUser.get("guid")
```

@WebDevFromScratch Could you unify this function and the `userReshare` function? I'd prefer the version of the `userReshare` function because here you don't return anything if the author is undefined. In `userReshare` we return false. Please also fix the issues Hound mentioned.